amazon-cloudwatch-agent.yaml

```diff
@@ -0,0 +1,60 @@
+package:
+  name: amazon-cloudwatch-agent
+  version: 1.300044.0
+  epoch: 0
+  description: CloudWatch Agent enables you to collect and export host-level metrics and logs on instances running Linux or Windows server.
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - go
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/aws/amazon-cloudwatch-agent
+      tag: v${{package.version}}
+      expected-commit: 7cf4fea44849bc2d9f6d9d4fd8284d447830850f
+
+  - uses: go/bump
+    with:
+      deps: github.com/opencontainers/runc@v1.1.14
+
+  - runs: |
```

Comment:
Any reason we don't use our go/build pipeline here?

Reply:
Not anything in particular RJ, using existing Make targets looked simpler to me so I proceeded with the same.

Reply:
I'm not sure about using `go/build` here since they might introduce additional stuff in the `amazon-cloudwatch-agent-linux` make target, we would miss it. 🤔